### PR TITLE
js_task-move-spider-DOM: add test, change readme

### PR DESCRIPTION
**Pull request by brainwashers**

### CODE REVIEW

File: cypress/integration/page.spec.js [hunk 1]
```diff
@@ -1,8 +1,43 @@
 'use strict';
 
-// TODO: edit it
-describe('Page', () => {
-  it('should be visitable', () => {
+Cypress.Commands.add('clickOnWall',
+  (position, xCoord, yCoord) => {
+    // NOTE: waiting for transition
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.get('.wall').click(position).wait(1000);
```
Comment (roman-mate) [inline cypress/integration/page.spec.js:7]: Based on documentation this should not be needed: https://docs.cypress.io/guides/core-concepts/interacting-with-elements.html#Animations

File: cypress/integration/page.spec.js [hunk 2]
```diff
@@ -1,8 +1,43 @@
...
+      const spiderTop = spider.offsetTop;
+      const spiderLeft = spider.offsetLeft;
+
+      expect(spiderTop).to.equal(xCoord);
+      expect(spiderLeft).to.equal(yCoord);
+    });
+  });
+
+describe('Spider', () => {
+  beforeEach('Open site', () => {
```
Comment (roman-mate) [inline cypress/integration/page.spec.js:20]: where is the message 'Open site' displayed?

File: cypress/integration/page.spec.js [hunk 3]
```diff
@@ -1,8 +1,43 @@
...
+      const spider = $spider.get(0);
+      const spiderTop = spider.offsetTop;
+      const spiderLeft = spider.offsetLeft;
+
+      expect(spiderTop).to.equal(xCoord);
+      expect(spiderLeft).to.equal(yCoord);
+    });
+  });
+
+describe('Spider', () => {
```
Comment (roman-mate) [inline cypress/integration/page.spec.js:19]: missing test cases when clicking outside of the rectangle.

File: cypress/integration/page.spec.js [hunk 4]
```diff
@@ -1,8 +1,43 @@
...
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.get('.wall').click(position).wait(1000);
+
+    cy.get('.spider').then(($spider) => {
+      const spider = $spider.get(0);
+      const spiderTop = spider.offsetTop;
+      const spiderLeft = spider.offsetLeft;
+
+      expect(spiderTop).to.equal(xCoord);
+      expect(spiderLeft).to.equal(yCoord);
```
Comment (roman-mate) [inline cypress/integration/page.spec.js:15]: expects should be in the test case.

**Review by roman-mate — CHANGES_REQUESTED** [review]

File: cypress/integration/page.spec.js (hunk 1 above)
Comment (brainwashers) [reply cypress/integration/page.spec.js:7]: yes, but we need this in that particular case. we have consulted with Yurii

File: cypress/integration/page.spec.js (hunk 2 above)
Comment (brainwashers) [reply cypress/integration/page.spec.js:20]: Nowhere. That’s just for me
can remove

File: cypress/integration/page.spec.js (hunk 1 above)
Comment (roman-mate) [reply cypress/integration/page.spec.js:7]: discussed that we'll debug how to make it work.

**Review by iBrianWarner — APPROVED** [review]

**Review by roman-mate — APPROVED** [review]
Great job!

**Pull request merged**